scripts/us_eia/opendata/process/common.py

```diff
@@ -334,7 +512,9 @@ def process(dataset, dataset_name, in_json, out_csv, out_sv_mcf, out_svg_mcf,
 
             # Add to rows.
             rows = []
+            #print(time_series)
```

Comment:
Please remove the commented line from everywhere if it's not required.

Reply:
Removed